articles/api/authentication/_logout.md

```diff
@@ -1,16 +1,22 @@
 # Logout
 
 ```http
-GET https://${account.namespace}/v2/logout?
-  client_id=${account.clientId}&
-  returnTo=LOGOUT_URL
+GET https://${account.namespace}/oidc/logout?
+  post_logout_redirect_uri=LOGOUT_URL&
+  id_token_hint=ID_TOKEN_HINT
 ```
 
 ```shell
 curl --request GET \
-  --url 'https://${account.namespace}/v2/logout' \
+  --url 'https://${account.namespace}/oidc/logout' \
   --header 'content-type: application/json' \
-  --data '{"client_id":"${account.clientId}", "returnTo":"LOGOUT_URL"}'
+  --data-raw '
+  { 
+    "client_id":"${account.clientId}", 
+    "post_logout_redirect_uri":"LOGOUT_URL", 
+    "id_token_hint":"ID_TOKEN_HINT",
+    "logout_hint":"LOGOUT_HINT"
```

Comment:
Even though it is permitted(*), there is no use case for "id_token_hint" & "logout_hint" to be included in the same request. A user needs one (normally "id_token_hint") or the other ("logout_hint" if the app cannot store IDTs)

I suggest we make two examples, each with one of the claims. Or just one with the "id_token_hint" and add a comment to the alternate "logout_hint"

(*) If they are both included their sid values must match (and would hence be redundant)

Reply:
I've removed the `logout_hint` from the examples. Unfortunately, there isn't really a way of adding in extra examples on this page. Perhaps it can go on the actual documentation page?